### PR TITLE
DSD-430: Using DS spacing values for the Heading component

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-430](https://jira.nypl.org/browse/DSD-430)

## This PR does the following:
- Updating the "em" spacing values for the Heading component with proper Chakra NYPL-theme values.

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr691-nlxj6vzp3jwhgqxwhdapxmqthelccscb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/614ce117d7000ac44349b82b

File: src/theme/components/heading.ts [hunk 1]
```diff
@@ -1,39 +1,46 @@
+const margins = {
+  marginTop: 0,
+  marginLeft: 0,
+  marginRight: 0,
+  marginBottom: 2, // var(--space-xs)
```
Comment (bigfishdesign13) [inline src/theme/components/heading.ts]: After seeing how this responded, this should be bumped up to `marginBottom: 4` to align with `--space-s`.

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
Just need to increase the spacing.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr691-nlxj6vzp3jwhgqxwhdapxmqthelccscb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/614ce117d7000ac44349b82b

**Review by bigfishdesign13 — APPROVED** [review]
The changes look good!

**Pull request merged**